sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificateClientTests.cs

```diff
@@ -61,6 +73,53 @@ public void DeleteIssuerArgumentValidation()
             Assert.AreEqual("issuerName", ex.ParamName);
         }
 
+        [Test]
+        public void SetContactsArgumentValidation()
+        {
+            ArgumentException ex = Assert.ThrowsAsync<ArgumentNullException>(() => Client.SetContactsAsync(null));
+            Assert.AreEqual("contacts", ex.ParamName);
+
+            // Below test failed since SetContactsAsync() method doesn't assert empty.
+            // Consider assert contacts empty scenario ?
+            List<CertificateContact> contacts = new List<CertificateContact>();
+            ex = Assert.ThrowsAsync<ArgumentNullException>(() => Client.SetContactsAsync(contacts));
+            Assert.AreEqual("contacts", ex.ParamName);
+        }
+
+        [Test]
+        public void GetCertificatePolicyArgumentValidation()
+        {
+            ArgumentException ex = Assert.ThrowsAsync<ArgumentNullException>(() => Client.GetCertificatePolicyAsync(null));
+            Assert.AreEqual("certificateName", ex.ParamName);
+
+            ex = Assert.ThrowsAsync<ArgumentException>(() => Client.GetCertificatePolicyAsync(string.Empty));
+            Assert.AreEqual("certificateName", ex.ParamName);
+            StringAssert.StartsWith("Value cannot be an empty string.", ex.Message);
+        }
+
+        [Test]
+        public void UpdateCertificatePolicyArgumentValidation()
+        {
+            CertificatePolicy policy = new CertificatePolicy { Exportable = true, ContentType = CertificateContentType.Pkcs12 };
+
+            ArgumentException ex = Assert.ThrowsAsync<ArgumentNullException>(() => Client.UpdateCertificatePolicyAsync(null, policy));
+            Assert.AreEqual("certificateName", ex.ParamName);
+
+            ex = Assert.ThrowsAsync<ArgumentException>(() => Client.UpdateCertificatePolicyAsync(string.Empty, policy));
+            Assert.AreEqual("certificateName", ex.ParamName);
+            StringAssert.StartsWith("Value cannot be an empty string.", ex.Message);
+
+            // Below test failed since UpdateCertificatePolicyAsync() method doesn't assert policy.
+            // Consider assert policy scenario ?
+            ex = Assert.ThrowsAsync<ArgumentException>(() => Client.UpdateCertificatePolicyAsync("test", null));
+            Assert.AreEqual("policy", ex.ParamName);
```

Comment:
Test failed, should assert policy argument null or empty ?

Reply:
Do what the docs say should be expected.